script/tool/test/util.dart

```diff
@@ -31,19 +32,11 @@ const String _defaultFlutterConstraint = '>=2.5.0';
 String getFlutterCommand(Platform platform) =>
     platform.isWindows ? 'flutter.bat' : 'flutter';
 
-/// Creates a packages directory in the given location.
-///
-/// If [parentDir] is set the packages directory will be created there,
-/// otherwise [fileSystem] must be provided and it will be created an arbitrary
-/// location in that filesystem.
-Directory createPackagesDirectory(
-    {Directory? parentDir, FileSystem? fileSystem}) {
```

Comment:
We no longer needed the `parentDir` version, so I was able to simplify this method quite a bit.